documentation/checker.md

```diff
@@ -1,6 +1,8 @@
 # Checker workflow (checker.wdl)
-This script runs vcf-to-gds-wf.wdl on a known set of inputs, and validates them against expected outputs with a quick md5sum check. On Terra, both inputs and outputs are pulled from the UCSC-owned topmed_workflow_testing bucket.
+This script runs pipelines on a known set of inputs, and validates them against expected outputs with a quick md5sum check. On Terra, both inputs and outputs are pulled from the UCSC-owned topmed_workflow_testing bucket.
 
 This workflow is not simply a copy of the check_gds step. Check_gds would not determine if the unique_variant_ids step ran successfully, instead, it a seperate validation check. Speaking of which, because the inputs are known to not take very long in check_gds, it is enabled *by default.* If you are replacing these files with your own sets of inputs and outputs, and that set is non-downsampled, modern TOPMed data, you may wish to consider skipping this step.
```

Comment:
Check_gds would not determine if the unique_variant_ids step ran successfully, instead, it **<missing verb??>** a **seperate** validation check.

1. Possibly a missing verb for the second half of this sentence?
2. Spelling error for "separate" highlighted in bold.

Reply:
Thanks for spotting this!